docs/user_guides/tls_proxy.md

```diff
@@ -419,16 +416,61 @@ openssl x509 -req -days 3650 -in client.csr -out client.pem -CA ca.pem -CAkey ca
 ```
 </details>
 
+The ```openssl.cnf``` file is a default configuration file of the openssl. If you want to know more details, visit [the manual page](https://www.openssl.org/docs/manmaster/man5/config.html). The ```gen.sh``` file can generate some related certificate files. And you can modify this file to customize for specific demands, such as the valid period of the certificate file, the length of the certificate key or the certificate file names.
```

Comment:
The `openssl.cnf` file is a default OpenSSL configuration file of the. See [manual page](https://www.openssl.org/docs/manmaster/man5/config.html) for more information. The `gen.sh` file generates relevant certificate files. You can modify the `gen.sh` file for different purposes such as changing the validity period of the certificate file, the length of the certificate key or the certificate file names.